introduction.md

```diff
@@ -1 +1,6 @@
 Hello, I'm from Ivanovo, Russia. My favorite CodeSchool path is Git path obviously, but JavaScript also not bad.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?